modules/nf-core/smncopynumbercaller/meta.yml

```diff
@@ -0,0 +1,35 @@
+name: "smncopynumbercaller"
+description: tool to call the copy number of full-length SMN1, full-length SMN2, as well as SMN2Δ7–8 (SMN2 with a deletion of Exon7-8) from a whole-genome sequencing (WGS) BAM file.
+keywords:
+  - copy number
```

Comment:
Could you add some more keywords here? (like input and output file types, some standard keywords people would search for this module by...)